lib/cache/index.js

```diff
@@ -9,16 +9,18 @@ function getInstanceModel (instance) {
     : instance.Model
 }
 
+function getModelPrimaryKeys (instance) {
+  return getInstanceModel(instance).primaryKeyAttributes.map((k) => instance[k])
+}
+
 function save (client, instance) {
   if (!instance) {
     return instance
   }
-
   const key = [
     getInstanceModel(instance).name,
-    instance.id
+    ...getModelPrimaryKeys(instance)
```

Comment:
This is really nice catch and will fix a problem with custom id name, but maybe let's also support composite primary keys? We just need to be sure if adapters handling more than 2 elements in this array. Lets add model with composite PK and see, but looks like VariableAdapter is not ready for this change https://github.com/DanielHreben/sequelize-transparent-cache-variable/blob/master/VariableAdaptor.js#L12